services/src/main/java/org/keycloak/services/resources/account/AccountRestService.java

```diff
@@ -464,6 +463,15 @@ private UserConsentModel createConsent(ClientModel client, ConsentRepresentation
         return consent;
     }
 
+    @Path("/applications")
```

Comment:
@douglaspalmer while I was reviewing your PR, just reading the sources I found out something similar to what you implemented https://github.com/keycloak/keycloak/blob/c6c5a427537b868640d31310505f7a6411bd861c/services/src/main/java/org/keycloak/services/resources/account/AccountRestService.java#L251

Do we really need the code for applications in both places? Maybe we could merge the logic of this code in a single place?